docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,217 @@
+to demonstrate how you can run the suite with only a few simple lines of code, 
+and see which kind of insights it can find.
+
+.. code-block:: bash
+
+    # Before we start, if you don't have deepchecks installed yet,
+    # make sure to run:
+    pip install deepchecks -U --quiet #--user
+"""
+
+#%%
+# Load Data and Prepare Data
+# ====================================================
+#
+# Load Data
+# -----------
+
+
+from deepchecks.tabular.datasets.classification import lending_club
+import pandas as pd
+
+data = lending_club.load_data(data_format='Dataframe', as_train_test=False)
+data.head(2)
+
+
+
+#%%
+# Split Data
+# -------------
+
+# convert date column to datetime, `issue_d`` is date column
+data['issue_d'] = pd.to_datetime(data['issue_d'])
+
+# Use data from June and July for train and August for test:
+train_df = data[data['issue_d'].dt.month.isin([6, 7])]
+test_df = data[data['issue_d'].dt.month.isin([8])]
+
+
+#%%
+# Run Deepchecks for Train Test Validation
+# ===========================================
+#
+# Define a Dataset Object
+# -------------------------
+#
+# Create a deepchecks Dataset, including the relevant metadata (label, date, index, etc.).
+# Check out :class:`deepchecks.tabular.Dataset` to see all of the columns that can be declared.
+
+
+#%%
+# Define Lending Club Metadata
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+categorical_features = ['addr_state', 'application_type', 'home_ownership', \
+  'initial_list_status', 'purpose', 'term', 'verification_status', 'sub_grade']
+index_name = 'id'
+label = 'loan_status' # 0 is DEFAULT, 1 is OK
+datetime_name = 'issue_d'
+
+
+#%%
+# Create Dataset
+# ^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+from deepchecks.tabular import Dataset
+
+# We explicitly state the categorical features,
+# otherwise they will be automatically inferred,
+# which may not work perfectly and is not recommended.
+
+# The label can be passed as a column name or
+# as a separate pd.Series / pd.DataFrame
+
+# all metadata attributes are optional.
+# Some checks require specific attributes and otherwise will not run.
+
+train_ds = Dataset(train_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+test_ds = Dataset(test_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+
+#%%
+
+# for convenience lets save it in a dictionary so we can reuse them for future Dataset initializations
+columns_metadata = {'cat_features' : categorical_features, 'index_name': index_name,
+                    'label':label, 'datetime_name':datetime_name}
+
+#%%
+# Run the Deepchecks Suite
+# --------------------------
+#
+# Validate your data with the :func:`deepchecks.tabular.suites.train_test_validation` suite.
+# It runs on two datasets, so you can use it to compare any two batches of data (e.g. train data, test data, a new batch of data
+# that recently arrived)
```

Comment:
example in parenthesis already given before and is redundant. Would leave the sentence though as it's worth to reiterate

Reply:
yes, kept it on purpose (incl. parenthesis) as I assume most (/almost all) people won't read both sections thoroughly and will skim (at least) one of them, so I think it's worth it (to help make the relevant use cases jump to their minds)